### PR TITLE
Fix an invalid engine handle error

### PR DESCRIPTION
**Pull request by bbrto21**
* To send something to the engine, the engine must be running.
* FlutterTizenView::SendInitialGeometry is used to send the geometry
  after running the engine

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

Fixes https://github.com/flutter-tizen/flutter-tizen/issues/361.

### CODE REVIEW

**Comment by bbrto21** [comment]
Fix : https://github.com/flutter-tizen/flutter-tizen/issues/361

File: shell/platform/tizen/flutter_tizen_view.cc [hunk 1]
```diff
@@ -63,8 +63,6 @@ void FlutterTizenView::SetEngine(std::unique_ptr<FlutterTizenEngine> engine) {
   text_input_channel_ = std::make_unique<TextInputChannel>(
       internal_plugin_registrar_->messenger(),
       std::make_unique<TizenInputMethodContext>(window_.get()));
-
-  OnRotate(window_->GetRotation());
```
Comment (swift-kim) [inline shell/platform/tizen/flutter_tizen_view.cc:67]: Could you check if the rotation works correctly without this line when an app is **started** in **landscape** mode on **mobile**?

Comment (bbrto21) [reply shell/platform/tizen/flutter_tizen_view.cc:67]: @swift-kim Of course :) I checked and it works fine.

**Review by swift-kim — APPROVED** [review]

**Comment by bbrto21** [comment]
@JSUYA As you already mentioned, I also don't think that's the scope of this PR. So if you really think it's necessary, you can open the PR yourself.

**Comment by swift-kim** [comment]
@bbrto21 @JSUYA That looks like a bug we currently have. `FlutterDesktopViewCreateFromNewWindow` should return `nullptr` if `RunEngine` fails. That is,

```cpp
if (!view->engine()->IsRunning()) {
  if (!view->engine()->RunEngine()) {
    return nullptr;
  }
}
```

**Comment by bbrto21** [comment]
> @bbrto21 @JSUYA That looks like a bug we currently have. `FlutterDesktopViewCreateFromNewWindow` should return `nullptr` if `RunEngine` fails. That is,

@JSUYA Would you please fix this bug?

**Pull request merged**